0.data_analysis/construct_phenotypic_expression_plate_4_fs_data/median_correlation_relationships/post_fs_aggregation_correlations/README.md

```diff
@@ -0,0 +1,24 @@
+# Aggregate Correlation Analysis
+In this analysis, the effect of different treatments on aggregated cell expression were compared, these treatments can be summarized as follows:
```

Comment:
```suggestion
In this analysis, we compare the effect of different treatments on aggregated cell morphology profiles using correlation.
We summarize these treatments as follows:
```

Minor edits to move to active voice (instead of passive voice) and to clarify

Reply:
That makes sense, just made the adjustments